CHANGES

```diff
@@ -1,3 +1,14 @@
+0.2.0 -> 0.3.0:
+	- Tizen Public C-API Single/Pipeline RC1 Fixed for 5.5 M2
```

Comment:
I want you to append a document on the major changes.
* https://github.com/nnsuite/nnstreamer/tree/master/Documentation

Reply:
Uh.. let's update Documentation in another thread.

Reply:
Addressed in #1638